cmd/init_tf_module_promise.go

```diff
@@ -150,7 +156,7 @@ func generateTerraformModuleResourceConfigurePipeline() (string, error) {
 					"containers": []any{
 						v1alpha1.Container{
 							Name:  "terraform-generate",
-							Image: "ghcr.io/syntasso/kratix-cli/terraform-generate:v0.2.0",
+							Image: "ghcr.io/syntasso/kratix-cli/terraform-generate:v0.4.0",
```

Comment:
this was never updated to the latest image 0.3.1 😢 pre-empting the next release of this by adding v0.4.0